imap_processing/ena_maps/utils/coordinates.py

```diff
@@ -0,0 +1,17 @@
+"""Define utils and classes related to coordinates of the ENA maps."""
+
+from enum import Enum
+
+
+class CoordNames(Enum):
+    """Enumeration of the names of the coordinates in the L1C and L2 ENA datasets."""
+
+    TIME = "epoch"
+    ENERGY = "energy_bin_center"
+    HEALPIX_INDEX = "healpix_pixel_index"
+
+    # The names of the az/el angular coordinates may differ between L1C and L2 data
```

Comment:
Why would they differ? Here they are identical, so this seems like it would be confusing.

Reply:
I don't know if they should differ conceptually, but given that the ENA instruments are still determining all their L2 data structures, names, shapes, etc. - I thought allowing them to be defined independently but all in this enum was the safest approach.